tests/unit/serve/instrumentation/conftest.py

```diff
@@ -116,4 +119,5 @@ def _ticker(self) -> None:
             self.collect(timeout_millis=self._export_interval_millis)
 
     opentelemetry.sdk.metrics.export.PeriodicExportingMetricReader = PatchedTextReader
+
     return collect_metrics, read_metrics
```

Comment:
Would putting a yield here and switching the PeriodicExportingMetricReader back to the Real implementation work? Seems like it would be better just in case later code tries to access the PeriodicExportingMetricReader without this fixtures in later tests.